tests/devices/test_default_mixed.py

```diff
@@ -99,11 +108,18 @@ def test_state_after_twoqubit(self, nr_wires, op, tol):
 
         assert np.allclose(dev.state, current_state, atol=tol, rtol=0)
 
-    @pytest.mark.parametrize("op", [AmplitudeDamping, DepolarizingChannel])
+    @pytest.mark.parametrize(
```

Comment:
Nice adjustment of the tests here! :100:

Reply:
Thanks! =D